scripts/core/menu/styles/menu.scss

```diff
@@ -323,6 +340,42 @@
     }
 }
 
+.top-menu-dropdown-button {
+    padding-left: 1.6rem;
+    padding-right: 1.6rem;
+    z-index: 2;
+    color: $white;
+    background-color: rgba(255,255,255,0.06);
+    transition: background-color ease-in-out 0.2s;
+    &:hover {
+        background-color: rgba(255,255,255,0.1);
+    }
+    &--active {
```

Comment:
Can you avoid dynamic class name generation? It's much easier to find usages of class names when you can search for the class name you see in the markup.

Reply:
We use BEM when writing CSS classes so it's always possible to search by block name (e.g. top-menu-dropdown-button in this case)

Reply:
Checking by block makes lookup take a few minutes instead of a few seconds. While it's possible, it's really inefficient.